client/components/marketing-survey/cancel-jetpack-form/index.tsx

```diff
@@ -308,80 +235,48 @@ const CancelJetpackForm: React.FC< Props > = ( {
 		setSurveyAnswerText( answerText );
 	};
 
-	const onGetCancellationOffer = () => {
-		recordEvent( 'calypso_purchases_cancel_get_discount' );
-	};
-
 	/**
 	 * Render the dialog buttons for the current step
 	 */
 	const renderStepButtons = () => {
-		const { disableButtons } = props;
+		const { disableButtons, cancellationCompleted, cancellationInProgress } = props;
 		const disabled = disableButtons;
-		const loadingOffers = shouldProvideCancellationOffer && fetchingCancellationOffers;
-		const applyingOffer = shouldProvideCancellationOffer && applyingCancellationOffer;
 		const close = {
 			action: 'close',
-			disabled: disabled || applyingOffer,
-			isPrimary:
-				lastStep !== cancellationStep && steps.CANCELLATION_OFFER_STEP !== cancellationStep,
-			label: translate( "I'll keep it" ),
+			disabled: disabled,
+			isPrimary: cancellationCompleted ? false : lastStep !== cancellationStep,
+			label: translate( 'Close' ),
 		};
 		const next = {
 			action: 'next',
 			disabled: disabled || disableContinuation,
+			isPrimary: cancellationCompleted,
 			label: translate( 'Next step' ),
 			onClick: clickNext,
 		};
-		const cancelText =
-			flowType === CANCEL_FLOW_TYPE.REMOVE
-				? translate( 'Remove subscription' )
-				: translate( 'Cancel subscription' );
-		const cancellingText =
-			flowType === CANCEL_FLOW_TYPE.REMOVE ? translate( 'Removing' ) : translate( 'Cancelling' );
+
 		const cancel = (
 			<Button
-				disabled={ disabled || applyingOffer || disableContinuation }
-				busy={ disabled }
+				disabled={ disabled || disableContinuation }
+				busy={ cancellationInProgress }
 				onClick={ onSubmit }
 				primary
-				scary
+				scary={ ! cancellationCompleted }
 			>
-				{ disabled ? cancellingText : cancelText }
+				{ cancellationCompleted ? translate( 'Submit' ) : translate( 'Cancel subscription' ) }
```

Comment:
Nitpick: should I be able to click "Submit" on the survey if I have not chosen an answer?

![Screenshot 2025-06-27 at 5 59 25 PM](https://github.com/user-attachments/assets/e5db0d19-d713-42c5-b408-da3e01d5c882)